ext/standard/string.c

```diff
@@ -5123,6 +5123,7 @@ PHPAPI size_t php_strip_tags_ex(char *rbuf, size_t len, uint8_t *stateptr, const
 			break;
 		case '<':
```

Comment:
Would it be crazy to encode < and > rather than merely allow them?  XHTML and HTML4/5 agree that &lt; &gt; are acceptable forms of <> in attributes.

Reply:
I don't think that would be the responsibility of strip_tags. So I think if the documentation promises that attributes won't be altered, the function should just allow them.

Reply:
As strip_tags is commonly misused as a security mechanism, I think it is best to err on the side of caution here and encode < and >. This will also limit the collateral damage if the attribute handling is in some way incorrect.

Reply:
Hmm, before I go crazy on this, the return value of `php_strip_tags_ex` is the new length, and the string is not expected to grow. So I'm a bit uncertain here, change the return value to a new string, which would mean touching all locations where strip_tags is used (in sanitizing_filters.c, file.c and filters.c). Or do you see an alternative way to approach this?

Reply:
Okay, I'm not sure if the direction I went is the smartest solution, but I just figured let's go a certain route and see what you think of this. My alternative solution was to modify the rbuf parameter to char**.